app/views/books/show.html.erb

```diff
@@ -0,0 +1,9 @@
+Book Title: <%= @book.title %>
+<% ratings = @book.ratings %>
+<% average_ratings = ratings.map(&:rating).sum/ratings.count %>
```

Comment:
You'd want to use the ActiveRecord method average(:column_name) to calculate this.